docs/content.zh/docs/dev/table/sql/jar.md

```diff
@@ -68,22 +70,22 @@ Flink SQL> REMOVE JAR '/path/hello.jar';
 ADD JAR '<path_to_filename>.jar'
 ```
 
-Currently it only supports to add the local jar into the session classloader.
+目前只支持将本地 jar 添加到会话类类加载器（session classloader）中。
 
 ## REMOVE JAR
```

Comment:
link tag?

Reply:
Sorry, I thought the title was in English, so I didn't need to write the link tag.
(The link tag is automatically generated when the title is in English.)
I've added the link tag .
Please help me review it again~Thank you ~